core/commands/files.go

```diff
@@ -567,6 +632,9 @@ Examples:
 					fmt.Fprintf(w, "%s\n", o.Name)
 				}
 			}
+			if out.Path != "" {
+				fmt.Fprint(w, "\n")
```

Comment:
Ideally we should skip this for the last entry we list. What would be the cleanest way to signal that? Closing the response?